projects/sbb-esta/angular-business/notification/src/notification/notification.component.ts

```diff
@@ -0,0 +1,188 @@
+  get typeSuccess(): boolean {
+    return this.type === NotificationType.SUCCESS;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-info')
+  get typeInfo(): boolean {
+    return this.type === NotificationType.INFO;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-error')
+  get typeError(): boolean {
+    return this.type === NotificationType.ERROR;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-warn')
+  get typeWarn(): boolean {
+    return this.type === NotificationType.WARN;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-toast-top-left')
+  get positionTopLeft(): boolean {
+    return this.toastPosition === NotificationToastPosition.TOPLEFT;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-toast-top-right')
+  get positionTopRight(): boolean {
+    return this.toastPosition === NotificationToastPosition.TOPRIGHT;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-toast-bottom-left')
+  get positionBottomLeft(): boolean {
+    return this.toastPosition === NotificationToastPosition.BOTTOMLEFT;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-toast-bottom-right')
+  get positionBottomRight(): boolean {
+    return this.toastPosition === NotificationToastPosition.BOTTOMRIGHT;
+  }
+
+  /** Type of notification. */
+  @Input()
+  type: 'success' | 'info' | 'error' | 'warn' = NotificationType.SUCCESS;
+
+  /** Type of notification. */
+  @Input()
+  toastPosition: 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
+
+  /** Type of notification. */
+  @Input()
+  readonly = false;
+
+  /** @docs-private */
+  @ViewChild('error', { read: TemplateRef, static: true })
+  errorIcon: TemplateRef<any>;
+
+  /** @docs-private */
+  @ViewChild('check', { read: TemplateRef, static: true })
+  checkIcon: TemplateRef<any>;
+
+  /** @docs-private */
+  @ViewChild('info', { read: TemplateRef, static: true })
+  infoIcon: TemplateRef<any>;
+
+  /** The icon to be used into the notification left side.
+   *  By default uses three icons for SUCCESS, ERROR or INFO notification type,
+   *  but the user can use his own icon using the NotificationIconDirective.
+   */
+  @Input()
+  @ContentChild(IconDirective, { read: TemplateRef, static: false })
+  set icon(notificationIcon: TemplateRef<any>) {
+    this._icon = notificationIcon;
+  }
+
+  get icon() {
+    if (this._icon) {
+      return this._icon;
+    }
+    switch (this.type) {
+      case NotificationType.SUCCESS:
+        return this.checkIcon;
+      case NotificationType.ERROR:
+        return this.errorIcon;
+      case NotificationType.INFO:
+        return this.infoIcon;
+      case NotificationType.WARN:
+        return this.errorIcon;
+      default:
+        return null;
+    }
+  }
+
+  private _icon: TemplateRef<any>;
+
+  /** Title displayed into the notification content */
+  @Input() title: string;
```

Comment:
Title does not exist https://digital.sbb.ch/de/webapps/components/notification

Reply:
True, I added this because of the comment on #83 "We currently use a title, text and a link.". Shall I remove this?

Reply:
Yes. For future reference, only consider the root issue in specifications. The comments will state potential additional requirements and if we consider them valid, the root issue will be updated.

Reply:
Fixed